benchmark/benchmark.go

```diff
@@ -47,7 +47,8 @@ import (
 	"google.golang.org/grpc/grpclog"
 )
 
-func newPayload(t testpb.PayloadType, size int) *testpb.Payload {
+// allows reuse of the same testpb.Payload object
```

Comment:
nit: Update the comments to look like complete sentences; Capital letter in the beginning and a period in the end.